pages/common/qmmp.md

```diff
@@ -0,0 +1,32 @@
+# qmmp
+
+> A small and light audio player.
```

Comment:
Could we make this command description more objective?
```suggestion
> A small and light audio player.
```

Reply:
> Could we make this command description more objective?

Agreed

Reply:
```suggestion
> An audio player.
```